internal/main

```diff
@@ -510,8 +510,24 @@ function post_build_preview {
 
 # Publishes the docs for Jenkins or for a dev.
 function publish {
-    [ -n "$JENKINS_URL" ] && activate_jenkins_env
-    echo "where does publish live? in docker? or just here, like it is in fe lambda?"
+    if [ -n "$JENKINS_URL" ]; then
+	activate_jenkins_env
+	MESSAGE="Test, build, and publish with Jenkins (build # $BUILD_NUMBER)"
+    else
+	MESSAGE="Manually published by $USER"
+    fi
+    git worktree add dragons gh-pages
+    rm -rf dragons/*
+    rsync -a docs/_build/html/ dragons/
+    cd dragons
+    touch .nojekyll
+    git add .
+    git commit -m "$MESSAGE"
+    git push origin gh-pages
+    cd -
+    # Can't use `worktree remove` until a newer version of git is available on Jenkins
```

Comment:
Can you not source the newer version of github to use the `worktree remove` or is that not in 2.29?

Reply:
No, I tried it. At one point I was activating the newer git. I guess it got left behind. Maybe it's a good idea to keep doing it, just in case.

Reply:
It probably can't hurt. I'll leave it up to you if want to make the change now or later.